versions-maven-plugin/pom.xml

```diff
@@ -27,6 +27,20 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.codehaus.mojo.versions</groupId>
+      <artifactId>versions-enforcer</artifactId>
+      <version>${project.version}</version>
+      <scope>runtime</scope>
+    </dependency>
```

Comment:
will be on plugin dependencies and will be loaded always, please remove
it test should be in versions-enforcer module